### ISSUE TITLE
Request: Curly Brace {{ }}, {% %}, {!! !!} ligatures.

### ISSUE THREAD

**Issue by Orillian** [kiliman/operator-mono-lig#38]
Double curly braces are used a lot in Angular, Mustache, Blade and Twig templates as output delimiters. 

**{{ variable }}** or **{{ expression result }}**
Then you have the others like the 
**{% twig execute delimiter %}** 
and the 
**{!! blade non-escaped variable !!}**

The double curly brace especially gets used in a bunch of other places, a lot of template syntax's in Javascript use them and I've also heard rumor of a dreaded use in Java, but since I don't do Java I'm OK with it.

**Comment by kiliman** [comment]
Ok, but how would you like the ligatures represented?

Is it simply moving the braces closer together and centering? Or would you prefer the left side to be pushed more to the left and vice versa?

I looked at Fira Code for inspiration, but they don't have those ligatures defined.

Can you mock up something in a paint program and show what you would like to see?

Thanks!

**Comment by Orillian** [comment]
Sure, I'll play around with a couple ideas. 
Since these are not common it might be a good idea to make their inclusion optional in the script if that is easy enough to do?

**Comment by kiliman** [comment]
Yes, I've already started preliminary work on optional ligatures. You'll even be able to generate multiple fonts so you can have language specific fonts for example.

**Comment by Orillian** [comment]
Awesome, I have the rest of this week off starting Wednesday, I'll be sitting around the house all day for a week, so I've got it in my calendar to draft up some options for these ligatures, as well as some of the others like the PHP space ship operator.

**Comment by Orillian** [comment]
Now that I'm finally feeling well enough to do so, I started playing around with these as well as a couple of the other ligatures we have in the issue list here!

Anyway I've got a couple blocks of png's that I'll drop in here hopefully tonight with a few options for some of these ligatures. For the most part moving them closer together works well, but in some cases actually touching the glyphs together looks pretty decent! 

I've been trying to use Font forge since I'm on windows but it's a PITA. On my desktop it breaks due to multi monitor setup and it crashes on my laptop. I was hoping to be able to learn a bit more about how this works so I could be a bit more helpful, but for now I'm stuck trying to find an alternative font editor for windows.

**Comment by Orillian** [comment]
![curly brace ligatures](https://user-images.githubusercontent.com/823192/35844787-76d88e66-0acc-11e8-899f-d0c35e032768.png)

Here are the first close together samples. Merging the double curly into a more uniform mustache shape might be nice if we can make that happen, but just touching them together leaves some funky edges.

**Comment by amitkparekh** [comment]
For the `{{` ligature, how does this look for a possible implementation? 

![image](https://user-images.githubusercontent.com/7276308/67866698-40bafe80-fb21-11e9-8d02-c0d2ede82185.png)

The two braces are closer together, and the right one is slightly (10%) smaller than left one, so it fits a nicely within the other brace.

**Comment by Orillian** [comment]
Honestly I like it! It looks a lot cleaner than my test idea above! 👍 

Sorry for the slow response, I've been away from here for a while now.
